packages/client/src/generation/generateClient.ts

```diff
@@ -133,6 +133,7 @@ export async function buildClient({
     types: 'index.d.ts',
     browser: 'index-browser.js',
     exports: clientPackageExports,
+    version: clientVersion,
```

Comment:
Found that this was missing when writing the test.

Reply:
I had to put the PR in draft again. Somehow changing either `version` or `private` causes our functional type tests to fail. I don't understand how they influence each other.

Reply:
So, i figured that out and fixed. It's quite a wild thing: If we have multiple npm packages with the same name and version in the tree and they all contain the same namespace declaration, TS will merge those together into single namespace. Not having a `version` in generated `package.json` prevented that. Now, when we added the version, all `Prisma` namespace declaration from all of the tests got merged together, sometimes overriding each other's properties, sometimes producing intersections that result in `never`` and so on. 

I've fixed that in my commit here by appending sha256 of the schema file to a generated package name. We might also consider making generated package name configurable.